### DISCUSSION TITLE
Page Tree

### DISCUSSION THREAD

**Discussion by bajahranks** [WikiTransformationProject/wikitraccs-releases#33] [General]
I would like to try your page tree solution. It's the first thing "Higher Ups" asked after showing them the results of Wikitraccs migration.

**Comment by heinrich-ulbricht** [comment]
Sure! I'll get back to you with details tomorrow.

**Comment by heinrich-ulbricht** [comment]
@bajahranks I attached the proof-of-concept version of the SharePoint Framework solution here: https://github.com/WikiTransformationProject/wikitraccs-releases/releases/download/v0.1.5/wikitraccs-page-tree.sppkg That's the one used to create the GIF in the [blog post](https://www.wikitransformationproject.com/blog/2023/03/05/confluence-page-tree-in-sharepoint/).

Using a test environment, I suggest setting up a site collection app catalog in the migration target site. Then add the app to the migration target site. Site Contents should contain this:

![image](https://user-images.githubusercontent.com/3469970/224350836-44f3a1f8-bda4-4ef2-b4d0-454f1f0b97ea.png)

Then open one of the migrated pages. The bread crumb should show. If not it would be nice to hear back from you with a screenshot.

A refined version including installation instructions will be made available in the coming weeks.

**Reply by bajahranks** [reply]
Was it deleted? the download page gives a 404 page error

**Reply by heinrich-ulbricht** [reply]
Ah sorry, it has been updated and renamed. But the link behaves weird. It should download now, although opening a 404 page. If this doesn't work look at the assets section of this release: https://github.com/WikiTransformationProject/wikitraccs-releases/releases/tag/v0.1.5

**Reply by bajahranks** [reply]
Once again thank you for your great work. It works. See confirmation 
![image](https://user-images.githubusercontent.com/24802231/224798305-d5b8b780-e866-4d50-8b5d-2b87b7005e98.png)

**Reply by bajahranks** [reply]
@heinrich-ulbricht I made an observation that the page tree is not shown to people with guest access

**Reply by heinrich-ulbricht** [reply]
Could you please check that guests are allowed to access the tenant app catalog site (assuming you are using the tenant app catalog)? This is often not the case. Assets for the SPFx solution are loaded from there.

**Comment by heinrich-ulbricht** [comment]
@bajahranks Just a heads up that an advanced page tree solution is now available on Microsoft AppSource: https://appsource.microsoft.com/en-us/product/office/WA200005439

This one is compatible with migrated pages and their hierarchy, but also with plain SharePoint pages.

**Comment by bajahranks** [comment]
Thanks for letting me know. I'm waiting for the company to purchase your product. It was already agreed upon at a business meeting and they also loved the page tree. I have also sent them the link above. Once again thank you for your work.
